src/stemmablenote.ts

```diff
@@ -100,19 +100,19 @@ export abstract class StemmableNote extends Note {
     // if note is flagged, cannot shorten beam
     switch (this.duration) {
       case '8':
-        if (this.beam == undefined) length = 35;
+        if (this.beam === undefined) length = 35;
```

Comment:
Can we extract this:

```js
const beamIsUndefined = this.beam === undefined;
```

Also, why does case '8': look different from all the other cases?

Reply:
refactored, no visual differences